src/client/Client.cc

```diff
@@ -1519,6 +1519,7 @@ mds_rank_t Client::choose_target_mds(MetaRequest *req, Inode** phash_diri)
           mds = repmap.at(r);
         }
       } else if (in->fragmap.count(fg)) {
+	req->send_to_auth = true;
```

Comment:
this is wrong. send_to_auth means sends to inode's auth. for this case, request should be sent to dirfrag's auth